packages/atti-components/src/components/Stepper/styles/StyledLabel.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+const StyledLabel = styled.p`
```

Comment:
I don't really like the API of the component. For example this Label that we have to set 2 times and depending on the `middle` prop, it'll be only white.
Moreover the design works right now because you're "lucky", the margin top + font-size of your Label makes it center but if we change the size of the Step (width / height), it won't be centered anymore. You'd better center it on purpose, for example with [flexbox](https://www.w3schools.com/css/css3_flexbox.asp).

For the API, I think that the Label should be only the text that is under the Step. The content of the Step (here a number) can easily be a props (String or Node ?). If you have other ideas, I'm totally opened ;)

Reply:
I have taken your advice into account and now the internal label is defined by the props value and the external label by a usable tag.
Is that okay?

Reply:
👌 It's way better ! Good job !